specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/expressRouteCircuit.json

```diff
@@ -1639,6 +1639,7 @@
       ],
       "description": "Express Route Circuit Connection in an ExpressRouteCircuitPeering resource."
     },
+
```

Comment:
please revert, no need to change previous api version

Reply:
done